sdk/tests/conformance2/rendering/blitframebuffer-multisampled-readbuffer.html

```diff
@@ -0,0 +1,301 @@
+<body>
+<canvas id="canvas" width="8" height="8"></canvas>
+<div id="description"></div>
+<div id="console"></div>
+
+<script id="vshader" type="x-shader/x-vertex">#version 300 es
+in highp vec2 aPosition;
+in highp vec4 aColor;
+out mediump vec4 vColor;
+void main() {
+    gl_Position = vec4(aPosition, 0.0, 1.0);
+    vColor = aColor;
+}
+</script>
+
+<script id="fshader" type="x-shader/x-fragment">#version 300 es
+precision mediump float;
+out vec4 oColor;
+in vec4 vColor;
+void main() {
+    oColor = vColor;
+}
+</script>
+
+<script>
+"use strict";
+
+var wtu = WebGLTestUtils;
+description("This test verifies the functionality of blitFramebuffer with multisampled readbuffer and sRGB color buffer.");
+
+var gl = wtu.create3DContext("canvas", undefined, 2);
+
+var tex_draw = gl.createTexture();
+var fb0 = gl.createFramebuffer();
+var fb1 = gl.createFramebuffer();
+var rb0 = gl.createRenderbuffer();
+var rb1 = gl.createRenderbuffer();
+var fbo_draw = gl.createFramebuffer();
+var size = 256;
+var vao;
+var program;
+var positionLoc;
+var colorLoc;
+
+if (!gl) {
+    testFailed("WebGL context does not exist");
+} else {
+    testPassed("WebGL context exists");
+
+    init();
+
+    var filters = [gl.LINEAR, gl.NEAREST];
+    for (var ii = 0; ii < filters.length; ++ii) {
+        blitframebuffer_multisampled_readbuffer(gl.SRGB8_ALPHA8, gl.SRGB8_ALPHA8, filters[ii]);
+    }
+}
+
+function init() {
+    program = wtu.setupProgram(gl, ['vshader', 'fshader']);
+    positionLoc = gl.getAttribLocation(program, "aPosition");
+    colorLoc = gl.getAttribLocation(program, "aColor");
+    if (!program || positionLoc < 0 || colorLoc < 0) {
+        testFailed("Set up program failed");
+        return;
+    }
+    testPassed("Set up program succeeded");
+
+    var vao = gl.createVertexArray();
+    gl.bindVertexArray(vao);
+    gl.clearColor(0, 0, 0, 1);
+    gl.viewport(0, 0, size, size);
+}
+
+function blitframebuffer_helper(readbufferFormat, drawbufferFormat, filter) {
+    // Create draw framebuffer and feed 0 to draw buffer
+    gl.bindTexture(gl.TEXTURE_2D, tex_draw);
+    gl.texImage2D(gl.TEXTURE_2D, 0, drawbufferFormat, size, size, 0, gl.RGBA, gl.UNSIGNED_BYTE, null);
+    gl.bindFramebuffer(gl.DRAW_FRAMEBUFFER, fbo_draw);
+    gl.framebufferTexture2D(gl.DRAW_FRAMEBUFFER, gl.COLOR_ATTACHMENT0, gl.TEXTURE_2D, tex_draw, 0);
+    wtu.glErrorShouldBe(gl, gl.NO_ERROR, "setup draw framebuffer should succeed");
+
+    gl.blitFramebuffer(0, 0, size, size, 0, 0, size, size, gl.COLOR_BUFFER_BIT, filter);
+    wtu.glErrorShouldBe(gl, gl.NO_ERROR, "blitframebuffer should succeed");
+
+    // Read pixels for comparision
+    var pixels = new Uint8Array(size * size * 4);
+    gl.bindFramebuffer(gl.READ_FRAMEBUFFER, fbo_draw);
+    gl.readPixels(0, 0, size, size, gl.RGBA, gl.UNSIGNED_BYTE, pixels);
+    wtu.glErrorShouldBe(gl, gl.NO_ERROR, "readpixels should succeed");
+    return pixels;
+}
+
+function checkPixel(color, expectedColor) {
+  var tolerance = 15;
```

Comment:
Not quite sure about the tolerance. If I set the tolerance to 7(In some other tests, we set the tolerance to 7). It will fail on my Linux desktop with OGL 4.5. And the biggest diff is actually 8. 

So I set the tolerance to 15. I suppose the implementation of OGL 4.5 can resolve multi-sampled srgb and draw to a srgb image. Then the test can pass on Linux with OGL 4.5. 

BTW, it seems to be reasonable that the diff would become bigger if we enabled msaa.